src/components/Proposal/Create/PluginForms/PluginInitializeFields.tsx

```diff
@@ -24,25 +25,29 @@ const fieldView = (plugin: string, title: string, field: string) => (
     <Field
       id={field}
       name={`${plugin}.${field}`}
+      validate={validate}
+      type={type}
     />
   </div>
 );
 
 const GenesisProtocolFields = (paramsProp: string) => (
-  <div className={css.parameters}>
-    {fieldView(paramsProp, "Queued Vote Required Percentage", "queuedVoteRequiredPercentage")}
-    {fieldView(paramsProp, "Queued Vote Period Limit", "queuedVotePeriodLimit")}
-    {fieldView(paramsProp, "Boosted Vote Period Limit", "boostedVotePeriodLimit")}
-    {fieldView(paramsProp, "Pre-Boosted Vote Period Limit", "preBoostedVotePeriodLimit")}
-    {fieldView(paramsProp, "Threshold Const", "thresholdConst")}
-    {fieldView(paramsProp, "Quiet Ending Period", "quietEndingPeriod")}
-    {fieldView(paramsProp, "Proposing Reputation Reward", "proposingRepReward")}
-    {fieldView(paramsProp, "Voters Reputation Loss Ratio", "votersReputationLossRatio")}
-    {fieldView(paramsProp, "Minimum DAO Bounty", "minimumDaoBounty")}
-    {fieldView(paramsProp, "DAO Bounty Const", "daoBountyConst")}
-    {fieldView(paramsProp, "Activation Time", "activationTime")}
-    {fieldView(paramsProp, "Vote on behalf", "voteOnBehalf")}
-  </div>
+  <Form>
+    <div className={css.parameters}>
+      {fieldView(paramsProp, "Queued Vote Required Percentage", "queuedVoteRequiredPercentage", value => validators.vaildRange(value, 50, 100))}
+      {fieldView(paramsProp, "Queued Vote Period Limit", "queuedVotePeriodLimit", validators.validNumber)}
+      {fieldView(paramsProp, "Boosted Vote Period Limit", "boostedVotePeriodLimit", validators.boostedVotePeriodLimit)}
+      {fieldView(paramsProp, "Pre-Boosted Vote Period Limit", "preBoostedVotePeriodLimit", validators.validNumber)}
+      {fieldView(paramsProp, "Threshold Const", "thresholdConst", validators.thresholdConst)}
+      {fieldView(paramsProp, "Quiet Ending Period", "quietEndingPeriod", validators.validQuietEndingPeriod)}
+      {fieldView(paramsProp, "Proposing Reputation Reward", "proposingRepReward", validators.validNumber)}
+      {fieldView(paramsProp, "Voters Reputation Loss Ratio", "votersReputationLossRatio", validators.validPercentage)}
+      {fieldView(paramsProp, "Minimum DAO Bounty", "minimumDaoBounty", value => validators.greaterThanOrEqualTo(value, 0))}
```

Comment:
this should be greater than 0 .
also the error msg should say : `"Must be a number greater than 0" ` instead of `"Must be a number greater than or equal to 1"`